JobConfig/primary/TargetPiStopPreFilter.fcl

```diff
@@ -0,0 +1,13 @@
+//
+// Base configuration for prefiltering pions based on time
+// original author: Sophie Middleton, 2024
+//
+#include "Production/JobConfig/primary/StopParticleFilter.fcl"
```

Comment:
Is this the only place StopParticleFilter is used?

Reply:
currently I think so